site/content/en/docs/tutorials/telemetry.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Telemetry"
+linkTitle: "telemetry"
+weight: 1
+date: 2020-11-24
+---
+
+## Overview
+
+minikube provides telemetry suppport via [OpenTelemetry tracing](https://opentelemetry.io/about/) to collect trace data for `minikube start`.
+
+Currently, minikube supports the following exporters for tracing data:
+- [Stackdriver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/master/exporter/stackdriverexporter)
+
+Other exporters can be contributed by users as needed.
```

Comment:
change this to a better wording.

if you like to see other exporters please either create an issue (link) or submit a PR here

Reply:
Good point, I added a Contributing section to make this clear.